src/test/java/com/puppycrawl/tools/checkstyle/checks/imports/PkgImportControlTest.java

```diff
@@ -70,159 +72,169 @@ public void setUp() {
     @Test
     public void testLocateFinest() {
         assertEquals("Unexpected response", icRoot, icRoot
-                .locateFinest("com.kazgroup.courtlink.domain"));
+                .locateFinest("com.kazgroup.courtlink.domain", "Class"));
```

Comment:
should we use "Type" here ? as import could of Enum or Interface or Annotation.
https://docs.oracle.com/javase/specs/jls/se7/html/jls-7.html#jls-7.5.1

Reply:
Name doesn't specifically matter. I just named it so to follow some example and such that you might see in some type of demo. I renamed it to `MyClass` like in the other import control test.